src/views/Editor/EditorHeader.tsx

```diff
@@ -103,6 +106,9 @@ export const EditorHeader = ({ documentId, readOnly, readOnlyVersion }: { docume
   }, [deliverablePlanning, dispatch, documentId, history, state.viewRegistry, viewId, workflowStatus])
 
   const title = documentType === 'core/editorial-info' ? 'Till red' : 'Artikel'
+
+  const isReadOnlyAndUpdated = workflowStatus?.name !== 'usable' && readOnly
```

Comment:
For a split second, the "Gå till senaste versionen" button is visible when the Editor is loading, due to the `workflowStatus` being `undefined`.
An extra check seems to do the trick:
```suggestion
  const isReadOnlyAndUpdated = workflowStatus && workflowStatus?.name !== 'usable' && readOnly
```